ergast_py/models/time.py

```diff
@@ -0,0 +1,28 @@
+""" Time class """
+
+import datetime
+from dataclasses import dataclass
+
+
+@dataclass
+class Time:
```

Comment:
Can you change this branch to be looking at [develop](https://github.com/Samuel-Roach/ergast-py/tree/develop)?

You'd then be able to use the [BaseModel](https://github.com/Samuel-Roach/ergast-py/blob/develop/ergast_py/models/base_model.py) class and save you from writing these dunder methods. Sorry about this, I'm aware it's not currently obvious which branch the predominant development should be under, looking to fix this as part of https://github.com/Samuel-Roach/ergast-py/issues/15